test/unit-tests/GPUtests/Traversal.cxx

```diff
@@ -13,6 +13,7 @@
 #include <cstdio>
 #include <cfloat>
 #include <random>
+#include <vector>
```

Comment:
Because users are using it in their codes and I wanted to make sure it works.  It's only in the test.